ssh_key_rotator/server/server.py

```diff
@@ -82,6 +82,8 @@ async def stop(self):
             stderr=asyncio.subprocess.DEVNULL,
         )
         await kill_task.wait()
+        log_file = f"{get_user_path()}/ssh/sshd_log"
+        os.remove(log_file)
```

Comment:
this should either be in a `finally` or you should be using a context manager - the point is, for anything that MUST be destroyed you want to make sure it is, even if something like an exception occurs prior to the cleanup step